tox/config.py

```diff
@@ -299,6 +273,8 @@ def get_version_info(pm):
 
 
 class SetenvDict(object):
+    _DUMMY = object()
```

Comment:
does this needs to be an instance can't it be a class e.g. instead? seems we use it as a singleton and all classes are already singleton, so maybe just object?

Reply:
I just moved this to its appropriate scope, without questioning if this is at all the best way to handle this.